features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2499,6 +2499,9 @@
                 <Origin>{{origin}}</Origin>
             {% endfor %}
         </FIDO2TrustedOrigins>
+        <!-- Time allocated for the users to authenticate -->
+        <UserResponseTimeout>{{fido.user_response_timeout}}</UserResponseTimeout>
```

Comment:
What is the time unit? (eg. mins, seconds). There is a way do define time unit in J2 as I have heard. If not indicate in the comment.

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/3360/commits/c1450738df06e6901287350c2ce5d69962cc02df